### DISCUSSION TITLE
奢侈品 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#438] [General]
# 奢侈品 - Miao Yu | 于淼

https://yufree.cn/cn/2023/10/31/luxury/

**Comment by KexingWestRoad** [comment]
买个东西的心理建设也能讲的如此的高大上，还能有这么多思考，哈哈哈哈，🐮🐮🐮

**Comment by DayuGuo** [comment]
讨论消费主义 No，
想买游戏了 Yes！

**Comment by Kiki-2333-Zhang** [comment]
花260买了《惊奇》。到现在已经完了差不多20小时，平均一小时花费13块钱。嗯，很值！
